auth/config/database.php

```diff
@@ -74,7 +74,7 @@
             'charset' => 'utf8',
             'prefix' => '',
             'prefix_indexes' => true,
-            'schema' => 'public',
+            'schema' => 'auth',
```

Comment:
My understanding is that was the plan.

Auth is supposed to be another service, so it shouldn't be sharing a database with another service. 

Later it might be another database (it still could be here, as well). But a separate schema just seemed like the easiest way to mimic it for now.

@tristan-orourke can confirm

Reply:
Yeah, I guess another schema would work. I dealt with it by setting up another database on the same postgres server, and pointing to that one in the .env file. I think the code that makes that work just got merged into main with PR #277.